Docker/README.md

```diff
@@ -34,14 +34,14 @@ First usage steps:
 1. Acquire the Docker image. There are two options:
     1. Download the prebuilt images from [ualibraries DockerHub](https://hub.docker.com/r/ualibraries/)
```

Comment:
This is probably from Jeff's first version, but we've created an organization in Docker Hub for consistency purpose with github: 
https://hub.docker.com/u/ualbertalib/

Could you update the references in the doc please?